### PR TITLE
good luck reviewing this

### PR DESCRIPTION
**Pull request by alexd-332**
**Coord Simplification/Standardisation**
	- Removed TCU passing airways clearances at YBAF, YPPF, YSBK
	- Removed all taxi calls (except military airports)
	- 20nm ENR<>ENR change parameter changed to 50nm. 20nm Heads-up still used for BIK, ELW, and PIY Internal optional coord
	- Fixed Boundary coord (clarified its half the applicable lateral standard, 2.5nm for ENR, 1.5nm for TCU/ADC)
	- Auto Release added at YBCS. Now available at all major Radar towers
	- Wording of auto-release adjusted to remove confusing double negatives
	- Changed more coord examples from coldline to hotline (almost all heads-up now via hotline)
	- All arrival/overflier coord to class D towers now by 5 mins to boundary
	- All Class D towers (metro and regional) now just have Next call, and arrivals/overflier Heads-up call.
	- Removed differences between IAP and visual tracking. ENR/TCU will now coordinate approach intentions to Metro Ds, and ENR/TCU will clear aircraft for the approach. Approach clearance responsibility still lies with ADC at Regional Class Ds.
	
	
**Fixes**
	- General typos
	- Clarified "Arrival Coord" at Class Ds to be "Arrivals/Overfliers"
	- BK ADC / SY TCU departure coord
	- "Unrestricted" readback phraseology corrected
	- Removed ADC ownership of airspace at YPAD, YBBN, YMML, YPPH. As per IRL.
	- PH TCU / PIY std assignable back to F180 as per RL

**Additions**
	- VFR waypoints to abbreviations
	- YMML ATIS info
	- YPAD departures controller table
	- Missing Oceanic coord from ASP page
	- ARL: Jet aircraft must be cleared via BOREE STAR, non-jet aircraft must be cleared via MEPIL STAR
	- BIK: Jet aircraft must be cleared via RIVET STAR, non-jet aircraft must be cleared via ODALE STAR
	- Add notes to Adelaide AD/TCU for handling coastal flights
	- Add note to Adelaide AD for referencing taxiways
	- Add reference to military pages for RAAF TERMA doc (providing charts for military procedures like YPED STARs, etc)
	
**Deletions**
	- Lines that explained that you must separate aircraft in your airspace (duh)
	- References to when you can and can't use visual sep at YBCS (it's not explained anywhere how to do that). can be added back later
	- R574 Release, shouldnt be owned by WLM TCU
	- Lines explaining what 'next' coordination is

### CODE REVIEW

File: docs/terminal/melbourne.md [hunk 1]
```diff
@@ -203,98 +203,105 @@ The Standard assignable level from ENR to ML TCU is `A090`. All other levels mus
...
 
-*\*When Runway 09 is in use for arrivals, all Runway 16 departures shall be assigned heading 160.*
-
-When Runway 09 is in use for departures, ML TCU shall nominate a heading to ML ADC for use as a standard assignable heading.
+**When Runway 09 is in use for arrivals, all Runway 16 departures shall be assigned heading 160.
 
 ### ML TCU / EN ADC
+
+#### Start Clearance
+When an aircraft requests start clearance, the EN SMC controller shall coordinate with ML TCU to obtain the start clearance.
```
Comment (mattkelly4) [inline docs/terminal/melbourne.md:239]: Does this apply to all departures, or just departures into TCU class C?

File: docs/terminal/sydney.md [hunk 2]
```diff
@@ -21,8 +21,10 @@
 * [Additional requirements](#airspace-structural-arrangements) must be met prior to opening SRI as a stand-alone position.
 
 ## Airspace
-The Sydney TMA spans a 45nm Radius around SY DME from `SFC` to `F280`. SY TCU is responsible for the Sydney TMA, except:  
+The Vertical limits of the SY TCU are `SFC` to `F280`.  
+SY TCU is responsible for the Sydney TMA, except:  
 
+- SY CTR `SFC` to `A005`
```
Comment (mattkelly4) [inline docs/terminal/sydney.md:27]: From memory, I think we designated SY TWR airspace as most but not all of the CTR.  Might need to specify that here

**Review by mattkelly4 — APPROVED** [review]

**Comment by mattkelly4** [comment]
I'll delete my PR since you've incorporated it here

File: docs/terminal/melbourne.md (hunk 1 above)
Comment (alexd-332) [reply docs/terminal/melbourne.md:239]: this was already in beforehand, but basically if the aircraft requests start clearance, then EN SMC needs to ask ML TCU for the clearance. ERSA outlines the requirements for requesting start clearance (not that most pilots on vatsim would be following that anyway)

File: docs/terminal/sydney.md (hunk 2 above)
Comment (alexd-332) [reply docs/terminal/sydney.md]: fixed

File: docs/terminal/melbourne.md (hunk 1 above)
Comment (mattkelly4) [reply docs/terminal/melbourne.md:239]: Should we change this to only aircraft who will enter the TCU? I figure if an aircraft wants to exit the coffin into class G or remain in EN airspace, it doesn't make sense to bother ML with it

Comment (alexd-332) [reply docs/terminal/melbourne.md:239]: We could. I figure it was worded as such to account for the fact that pilots often won't follow the ERSA and request start clearance. It's more of a pilot-initiated coord than it is sector-initiated coord

**Review by mattkelly4 — APPROVED** [review]

**Review by LeGiraffe — APPROVED** [review]

**Pull request merged**